kapitan/cli.py

```diff
@@ -219,6 +219,13 @@ def main():
     init_parser.add_argument('--directory',
                              default=from_dot_kapitan('init', 'directory', '.'),
                              help='set path, in which to generate the project skeleton, assumes directory already exists. default is "./"')
+    init_parser.add_argument('--verbose', '-v', help='set verbose mode',
+                                action='store_true',
+                                default=from_dot_kapitan('init', 'verbose', False))
+    init_parser.add_argument('--target-name', help='provide a target-name ex: name1,name2',
+                                default=from_dot_kapitan('init', 'verbose', ""))
```

Comment:
`from_dot_kapitan('init', 'targets', []))`

Reply:
It is later converter to array in the initialiser object __init__ function.

Reply:
Yes I've noticed, but the correct behaviour would be to accept it as an array in the first place, see `compile --targets` arg